.gitignore

```diff
@@ -15,3 +15,4 @@ astroid.egg-info/
 .pytest_cache/
 .mypy_cache/
 venv
+.vscode/
```

Comment:
@Pierre-Sassoulas I saw an old discussion on `pylint` about this. Do we want to commit this to the project?

Reply:
reset and ignored locally